src/renderer/api/terminal-api.ts

```diff
@@ -144,18 +144,6 @@ export class TerminalApi extends WebSocketApi<TerminalEvents> {
     this.socket.binaryType = "arraybuffer";
   }
 
-  destroy() {
-    if (!this.socket) return;
-    const controlCode = String.fromCharCode(4); // ctrl+d
-
-    this.sendMessage({ type: TerminalChannels.STDIN, data: controlCode });
```

Comment:
We don't need to send control code anymore?

Reply:
This was supposed to be a followup to #4536.

We already clean up the PTY and the websocket either exits/closes. So I didn't think this was necessary. And from my testing it doesn't seem to be. But I might be missing something.